test/protocol_test.py

```diff
@@ -49,80 +55,106 @@ def test_deserialize_token():
 
 def test_serialize_token_binary():
     token = protocol.Token(valid_from=1365084334, valid_to=1365084634,
-                           username="noa")
+                           username=six.b("noa"))
     buf = protocol.Token.serialize(token)
-    assert buf == "t\x00\x00\x00Q]\x88\xaeQ]\x89\xda\x00\x00\x00\x03noa\x00"
+    assert buf == six.b(
+        "t\x00\x00\x00Q]\x88\xaeQ]\x89\xda\x00\x00\x00\x03noa\x00"
+    )
 
 
-def test_deserialize_token():
-    buf = "t\x00\x00\x00Q]\x8b\x17Q]\x8bU\x00\x00\x00\x04test"
+def test_deserialize_token_________rename_me():
```

Comment:
There were two tests with the name `test_deserialize_token()`, so this second one was shadowed. I put that name instead so that we would catch it in code review later. Success, yay!

What should I rename it to?